var/spack/repos/builtin/packages/r-rsubread/package.py

```diff
@@ -6,15 +6,16 @@
 from spack.package import *
 
 
-class BioconductorRsubread(RPackage):
+class RRsubread(RPackage):
     """Mapping, quantification and variant analysis of sequencing data"""
 
     homepage = "https://bioconductor.org/packages/3.16/bioc/html/Rsubread.html"
     url = "https://bioconductor.org/packages/release/bioc/src/contrib/Rsubread_2.14.2.tar.gz"
 
-    bioc = "rsubread"
+    bioc = "Rsubread"
 
     depends_on("r-matrix")
```

Comment:
for consistency to good practices, please move dependencies after the version statements